query_test.go

```diff
@@ -1151,20 +1151,20 @@ func TestQueryReadEmpty(t *testing.T) {
 	// Submit query
 	assert.Nil(t, query.Submit())
 
-	// Validate status, since query was used this is should be complete
+	// Validate status, since array was empty this is should be incomplete
 	status, err := query.Status()
 	assert.Nil(t, err)
-	assert.Equal(t, TILEDB_COMPLETED, status)
+	assert.Equal(t, TILEDB_INCOMPLETE, status)
 
 	// Validate query type
 	queryType, err := query.Type()
 	assert.Nil(t, err)
 	assert.Equal(t, TILEDB_READ, queryType)
 
-	// No results because query it empty
+	// Has results because array it empty
 	hasResults, err := query.HasResults()
 	assert.Nil(t, err)
-	assert.Equal(t, false, hasResults)
+	assert.Equal(t, true, hasResults)
```

Comment:
@Shelnutt2 has the behavior changed in the case of reading an empty array? I can see results in offsets after query submit, stepping at the var sized attribute data type size, actual values zero

Reply:
No behavior change